pkgs/applications/networking/syncthing/default.nix

```diff
@@ -3,19 +3,19 @@
 let
   common = { stname, target, postInstall ? "" }:
     buildGoModule rec {
-      version = "1.3.0";
+      version = "1.3.2";
       name = "${stname}-${version}";
```

Comment:
probably `name` can be removed here ?

Reply:
I'm not certain about that. `name` is constructed with `stname` which comes from a function parameter of the `common` function. Well... the function's parameter is the set, but... you know what I mean ;)

Reply:
yes, didn't see it was `stname` . `pname = stname;` could be a different way (or renaming stname to pname then inherit pname